apps/fumufumu-backend/src/auth.ts

```diff
@@ -1,32 +1,42 @@
 import { betterAuth } from "better-auth";
 import { drizzleAdapter } from "better-auth/adapters/drizzle";
-import type { DrizzleD1Database } from 'drizzle-orm/d1';
-import type { Env } from './index';
+import type { Env, DbInstance } from './index';
 
 /**
  * Drizzleインスタンスと環境変数を受け取って Better Auth インスタンスを生成する関数
- * @param db DrizzleD1Databaseインスタンス
+ * @param db DbInstance型
  * @param env Cloudflare Workersの環境変数 (Bindings)
  * @returns Better Authインスタンス
  */
-export function createBetterAuth(db: DrizzleD1Database, env: Env) {
+export function createBetterAuth(db: DbInstance, env: Env) {
 	return betterAuth({
 		database: drizzleAdapter(db, {
 			provider: "sqlite",
 		}),
+		emailAndPassword: {
+			enabled: true,
+			autoSignIn: true,
+		},
 		user: {
-			modelName: "auth_users",
+			modelName: "authUsers",
 		},
 		session: {
-			modelName: "auth_sessions",
+			modelName: "authSessions",
+			// 💡 パフォーマンス改善のための Cookie Cache を有効にする
+			cookieCache: {
+				enabled: true,
+				maxAge: 5 * 60, // 5分間キャッシュ
+			}
```

Comment:
[question]
どういう観点でキャッシュの時間を設定するのか観点がわからなかったのでそれを踏まえた質問です！
キャッシュを5分間にしたのは、どういう観点で設定しましたか？

Reply:
すみません、テキトーでしたmm
調べ直しましたが、いろんな観点で変わるっぽいです！
<img width="779" height="785" alt="image" src="https://github.com/user-attachments/assets/14628fc6-84eb-4457-b84b-67abb777864d" />


一週間でどうでしょうか？
毎日サインイン求められるのつらいですし、アプリ内をサーフィンするにも一週間は良いラインかなと。
```suggestion
			cookieCache: {
				enabled: true,
				maxAge: 7 * 24 * 60 * 60, // 一週間キャッシュ
			}
```